.github/workflows/ci.yml

```diff
@@ -147,34 +147,126 @@ jobs:
         run: node . run licenses
 
   smoke-tests:
-    name: Smoke Tests
+    # This cant be tested on Windows because our node_modules directory	
+    # checks in symlinks which are not supported there. This should be	
+    # fixed somehow, because this means some forms of local development	
+    # are likely broken on Windows as well.	
+    name: Smoke Publish - ${{ matrix.platform.name }} - ${{ matrix.node-version }}
     if: github.repository_owner == 'npm'
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - name: Linux
+            os: ubuntu-latest
+            shell: bash
+        node-version:
+          - 20.17.0
+          - 20.x
+          - 22.9.0
+          - 22.x
```

Comment:
What's the possibility we can keep this to just one node version? We don't need a full matrix on this, right?

Reply:
i fixes this only runs latest